api/desecapi/models.py

```diff
@@ -53,6 +55,9 @@ class Token(rest_framework.authtoken.models.Token):
     )
```

Comment:
`key` field `max_length` is too large for what's returned by `generate_key()`. (Adding here because I can only comment on line 53 and after)